src/plugins/intel_gpu/src/graph/fully_connected.cpp

```diff
@@ -161,6 +161,11 @@ std::vector<layout> fully_connected_inst::calc_output_layouts(fully_connected_no
     format::type output_format = is_static && !allow_new_shape_infer ? get_preferred_format(node, impl_param) :
                                               input_layout.format.value;
 
+    // Select preferred output format if the impl_type of FC node is onednn.
+    if (node.get_preferred_impl_type() == impl_types::onednn
+        && node.get_preferred_output_fmt() != format::any)
+        output_format = node.get_preferred_output_fmt();
+
```

Comment:
BTW, preferred_output_fmt is set not only for onednn, but also dynamic shape scenario. 
I think we can set output format with node.get_prererred_output_fmt if it is not empty or any, not only if it is onednn. How do you think?

Reply:
I agree but I'm not sure. I will change and then try jenkins test.

Reply:
Changed